packages/client/lib/client.ts

```diff
@@ -51,11 +58,13 @@ export default class EthereumClient extends events.EventEmitter {
       this.config.syncmode === 'full'
         ? new FullEthereumService({
             config: this.config,
-            db: options.db,
+            chainDB: options.chainDB,
+            stateDB: options.stateDB,
           })
         : new LightEthereumService({
             config: this.config,
-            db: options.db,
+            chainDB: options.chainDB,
+            stateDB: options.stateDB,
```

Comment:
Do we need a stateDB in LightEthereumService?

Reply:
No, you are right, I will remove. Will keep the member association in `EthereumService` though, might be more flexible for future use cases (respectively generally not such a big question and we can also move later to `FullEthereumService` if we decide to).